config/apiserver/deployment.yaml

```diff
@@ -51,6 +51,17 @@ spec:
           - --audit-webhook-config-file=$(AUDIT_WEBHOOK_CONFIG_FILE)
           - --audit-webhook-mode=$(AUDIT_WEBHOOK_MODE)
           - --audit-webhook-initial-backoff=$(AUDIT_WEBHOOK_INITIAL_BACKOFF)
+          - --feature-sessions=true
+          - --sessions-provider-group=identity.miloapis.com
+          - --sessions-provider-version=v1alpha1
+          - --sessions-provider-resource=sessions
```

Comment:
Are we able to remove these flags now that it's expected the sessions provider will always use Milo's public API?